superset/sql_lab.py

```diff
@@ -562,3 +582,23 @@ def execute_sql_statements(  # pylint: disable=too-many-arguments, too-many-loca
         return payload
 
     return None
+
+
+def cancel_query(query: Query, user_name: Optional[str] = None) -> None:
+    """Cancal a running query."""
```

Comment:
spelling nit: cancel

also please update the docblock to match all the others with arguments

Reply:
improved by 4706b4308ec4a82aa765aafaa70ad2057977b98b